app/connectors/httpParsers/PublishedProcessHttpParser.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors.httpParsers
+
+import models.errors.{InternalServerError, InvalidProcessError}
+import models.{PublishedProcess, RequestOutcome, ScratchResponse}
+import play.api.Logger
+import play.api.http.Status._
+import uk.gov.hmrc.http.HttpReads
+
+object PublishedProcessHttpParser extends HttpParser {
+
+  val logger: Logger = Logger(getClass)
+
+  implicit val publishedProcessHttpReads: HttpReads[RequestOutcome[PublishedProcess]] = {
+    case (_, _, response) if response.status >= 200 && response.status <= 299 =>
```

Comment:
would be clearer to use OK and MULTIPLE_CHOICES ??

Reply:
I think I'll just change it to `_ == OK` since that's what it's meant to be anyway